time stamp
float64[] x
float64[] y
string source_frame
string destination_frame

---

float64[] x_source
float64[] y_source
float64[] z_source

float64[] x_dest
float64[] y_dest
float64[] z_dest
